src/components/PeopleFilters.tsx

```diff
@@ -1,17 +1,97 @@
-export const PeopleFilters = () => {
+import cn from 'classnames';
+import { SearchLink } from './SearchLink';
+import { FilterBy } from '../types/FilterBy';
+import { useEffect, useState } from 'react';
+import { centuryList } from '../utils/centuryList';
+
+type PeopleFilterProps = {
+  searchParams: URLSearchParams;
+  setQuery: React.Dispatch<React.SetStateAction<string>>;
+  setSearchParams: React.Dispatch<React.SetStateAction<URLSearchParams>>;
+  activeQuery: string;
+  toggleCenturies: (currentCentury: number) => string[];
+  activeCenturies: string[];
+};
+
+export const PeopleFilters = ({
+  toggleCenturies,
+  searchParams,
+  setQuery,
+  activeQuery,
+  setSearchParams,
+  activeCenturies,
+}: PeopleFilterProps) => {
+  const [activeFilter, setActiveFilter] = useState('');
+
+  useEffect(() => {
+    setActiveFilter(searchParams.get('sex') || '');
+    setQuery(searchParams.get('query') || '');
+  }, [searchParams, setQuery]);
+
+  const handleQueryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const newSearchParams = new URLSearchParams(searchParams);
+
+    if (!event.target.value) {
+      newSearchParams.delete('query');
```

Comment:
use `enum QueryParams`
```suggestion
      newSearchParams.delete(QueryParams.Query);
```